docs/guides/scaling_crawlers.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: scaling-crawlers
+title: Scaling our crawlers
+description: To infinity and beyond!...within limits
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+
+import CodeBlock from '@theme/CodeBlock';
+
+import MinMaxConcurrencySource from '!!raw-loader!./scaling_crawlers_minMaxConcurrency.ts';
+import MaxRequestsPerMinuteSource from '!!raw-loader!./scaling_crawlers_maxRequestsPerMinute.ts';
+import AutoscaledPoolOptionsSource from '!!raw-loader!./scaling_crawlers_autoscaledPoolOptions.ts';
+
+As we build our crawler, we might want to control how many requests we do to the website at a time. Crawlee provides several options to fine tune how many parallel requests should be made at any time, how many requests should be done per minute, and how should scaling up and down happen based on the system resources available.
+
+:::tip
+
+All of these options are available on all crawlers Crawlee provides, but for this guide we'll be using the <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink>. We can see all options that are available <ApiLink to="cheerio-crawler/interface/CheerioCrawlerOptions">`here`</ApiLink>
+:::
+
+## `minConcurrency` and `maxConcurrency`
+
+These control how many parallel requests can be run at any time. By default, crawlers will start with one parallel request at a time and scale up over time to a maximum of 1000 requests at a time.
+
+:::caution Don't set `minConcurrency` too high!
+
+Setting this option too high compared to the available system resources will make our crawler run extremely slow or might even crash.
+
+It's recommended to leave it at the default value that is provided and letting the crawler scale up and down automatically based on available resources instead.
+
+:::
+
+<CodeBlock language="js">
+	{MinMaxConcurrencySource}
+</CodeBlock>
+
+## `maxRequestsPerMinute`
```

Comment:
I would place this above the min/maxConcurrency as it's a much better rate limiting option than those two.

I'm also thinking that maybe we should set some reasonable default, like 200 or something that does not overload a website 😄

Reply:
Makes sense. On the other hand, if we have `maxConcurrency` set to 1000 as the paragraph above states, maybe we should adjust that as well?

Reply:
We could change the default for maxConcurrency to 200 from 1000, I think that's a smart idea. I'll also move this above them as I agree with you